packages/sc-project-initializer/README.md

```diff
@@ -8,14 +8,14 @@ This package is meant to facilitate smart contract development.
 
 ## Repository Initialization
 
-Simply run the following command:
+Simply run the following command with your project name:
 
 ```shell
 npx @massalabs/sc-project-initializer init <projectName>
 ```
 
-Toolkit documentation is available [here](commands/init/README.md)
+The initializer documentation is available [here](https://github.com/massalabs/massa-sc-toolkit/blob/main/packages/sc-project-initializer/commands/init/README.md).
 
-## Develop
+## Contribute
 
-You can initialize a project with `npx sc-project-initializer init testDir`.
+For development purpose you can initialize a local project with `npx sc-project-initializer init <projectName>`.
```

Comment:
For development purposes, you can initialize a local project with `npx sc-project-initializer init <projectName>`.